backend/src/helper/index.js

```diff
@@ -11,6 +11,21 @@ export function findInObject(data, searchParameter, exclude){
     })
 }
 
+export function filterByRelation(data, searchParameter, relation){
+    return JSON.parse(JSON.stringify(data)).filter(function (parameter) {
+        return Object.keys(searchParameter).every(function (key) {
+            return relation(parameter[key], searchParameter[key]);
+        })
+    })
+}
+
+export function mergeObject(data, searchParameter1, searchParameter2) {
+    let resultsToMerge = filterObject(data, searchParameter1);
+    data = filterObject(data, searchParameter2);
+    data = data.concat(resultsToMerge);
+    return data;
+}
+
 export function fetchData(req) {
```

Comment:
I think we can all the data from the req.body so you can just do const {country, name , age , basically, anything from the request} = req.body, this way you will not need to reassign them again in a function. and I believe you can rename them if you want as well, like `const { name: fullName}`

Reply:
@Aeunal also, there is a conflict if you happy to fix it is fine otherwise I can go ahead and fix the conflict